example/app/config.yml

```diff
@@ -59,7 +59,7 @@ default:
       username: cumulususer
     services:
       EcsTaskHelloWorld:
-        image: cumuluss/cumulus-ecs-task:1.3.0
+        image: cumuluss/cumulus-ecs-task:1.4.0
```

Comment:
Not that anyone is deploying with Kes, but would this version be backwards compatible with a Kes deployment?

Reply:
Should be.   And in fact I'm using it in the backport.      What it's *not* compat with is node8 + kes.  

Of course that won't matter for more than a week or so.